charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -21,6 +21,9 @@
     {{- include "containers-common-env" . | nindent 4 }}
     - name: DD_LOG_LEVEL
       value: {{ .Values.agents.containers.systemProbe.logLevel | default .Values.datadog.logLevel | quote }}
+    - name: DD_SYSTEM_PROBE_HEALTH_PORT
```

Comment:
Is this always going to be present? There should be a check for the health port value being > 0 to add this env var.